<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QPushButton" name="pushSearch_on_name">
        <property name="text">
         <string>Search on name</string>
        </property>
        <property name="icon">
         <iconset resource="image.qrc">
          <normaloff>:/new/pref_image/images/search_data.png</normaloff>:/new/pref_image/images/search_data.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="actionSelect">
        <property name="text">
         <string>Show data</string>
        </property>
        <property name="icon">
         <iconset resource="image.qrc">
          <normaloff>:/new/pref_image/images/show_data.png</normaloff>:/new/pref_image/images/show_data.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButtonDelete_data">
        <property name="text">
         <string>Delete data</string>
        </property>
        <property name="icon">
         <iconset resource="image.qrc">
          <normaloff>:/new/pref_image/images/delete_data.png</normaloff>:/new/pref_image/images/delete_data.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QTableView" name="tableView"/>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QLineEdit" name="lineSearchName"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEditDelete">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>676</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_data"/>
    <addaction name="actionUpdate_data"/>
   </widget>
   <widget class="QMenu" name="menuAbout_author">
    <property name="title">
     <string>About </string>
    </property>
    <addaction name="actionAbout_the_author"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionAccept_window_size"/>
    <addaction name="actionReset_window_size"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionRussian"/>
    <addaction name="actionEnglish"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSettings"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuAbout_author"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionexit">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/pref_image/images/exit.png</normaloff>:/new/pref_image/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAdd_data">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/pref_image/images/add_data.png</normaloff>:/new/pref_image/images/add_data.png</iconset>
   </property>
   <property name="text">
    <string>Add data</string>
   </property>
  </action>
  <action name="actionUpdate_data">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/pref_image/images/update_data.png</normaloff>:/new/pref_image/images/update_data.png</iconset>
   </property>
   <property name="text">
    <string>Update data</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/pref_image/images/connect.png</normaloff>:/new/pref_image/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionAbout_the_author">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/pref_image/images/about.png</normaloff>:/new/pref_image/images/about.png</iconset>
   </property>
   <property name="text">
    <string>About the author</string>
   </property>
  </action>
  <action name="actionAccept_window_size">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/pref_image/images/accept_window_size.png</normaloff>:/new/pref_image/images/accept_window_size.png</iconset>
   </property>
   <property name="text">
    <string>Accept window size</string>
   </property>
  </action>
  <action name="actionReset_window_size">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/pref_image/images/reset_window_size.png</normaloff>:/new/pref_image/images/reset_window_size.png</iconset>
   </property>
   <property name="text">
    <string>Reset window size</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionexit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
